cmd/experimental/kueue-viz/backend/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Build stage
+FROM golang:latest as builder
+
+# Install necessary tools for building
+# RUN apk add --no-cache gcc musl-dev
+
+# Set the working directory
+WORKDIR /app
+
+# Copy Go modules manifests and download dependencies
+COPY go.mod go.sum ./
+RUN go mod download
+
+# Copy the application source code
+COPY . .
+
+# Build the application
+RUN go build -o app .
+
+# Runtime stage
+FROM golang:latest
```

Comment:
declare as `ARG BUILDER_IMAGE=golang:1.23` as here: https://github.com/kubernetes-sigs/kueue/blob/8cc315716e31d8439665452f8e5627136e352363/Dockerfile#L1.
One to align the versions, second to make it easy to find when we bump it.

Reply:
done